app/_src/deck/guides/apiops.md

```diff
@@ -188,8 +188,24 @@ Most commonly, you will use the commands from CI/CD tools built into your versio
 to bring full and partial {{site.base_gateway}} configurations together to create APIOps for your 
 particular needs.
 
+## Extending OpenAPI Specifications
+Several custom annotations can be used within the OpenAPI specification allowing you to declare {{site.base_gateway}} capabilities directly in the specification document. All custom annotations related to {{site.base_gateway}} configuration are prepended with ```x-kong-``` label.
+
+- ```x-kong-tags```: specify the tags to use for each Kong entity generated. The tags can be overridden when doing the conversion. This can only be specified on document level.
+- ```x-kong-service-defaults```: the defaults for the [Kong services](/gateway/{{page.release}}/admin-api/#service-object) generated from 'servers' object in the OpenAPI spec. These defaults can also be added to "path" and "operation" objects, in which case a new Service entity will be generated.
+- ```x-kong-upstream-defaults```: the defaults for [upstreams](https://docs.konghq.com/gateway/latest/admin-api/#upstream-object) generated from the `servers` object in the OpenAPI spec. These defaults can also be added to `path` and `operation` objects, in which case a new Service and Upstream entity will be generated.
+- ```x-kong-route-defaults```: the defaults for the [Kong routes](/gateway/{{page.release}}/admin-api/#route-object) generated from 'paths' in the OpenAPI spec.
+- ```x-kong-name```: the above directive gives the entire spec file its name. This will be used for naming the service and upstream objects in Kong. If not given it will use the 'info.title' field above to name them, or a random UUID if the 'info.title' field is missing. Names are converted into valid identifiers. This directive can also be used on "path" and "operation" objects to name them. Similar to operationId, each x-kong-name must be unique within the spec file.
```

Comment:
```suggestion
- `x-kong-name`: The name for the entire spec file. This is used for naming the service and upstream objects in {{site.base_gateway}}. If not given, it will use the `info.title` field to name these objects, or a random UUID if the `info.title` field is missing. Names are converted into valid identifiers. This directive can also be used on `path` and `operation` objects to name them. Similarly to `operationId`, each `x-kong-name` must be unique within the spec file.
```